modules/exploits/multi/php/car_rental_system.rb

```diff
@@ -0,0 +1,163 @@
+##
+# This module requires Metasploit: https://metasploit.com/download
+# Current source: https://github.com/rapid7/metasploit-framework
+##
+
+class MetasploitModule < Msf::Exploit::Remote
+  Rank = ExcellentRanking
+
+  include Msf::Exploit::Remote::HttpClient
+  include Msf::Exploit::Remote::HttpServer
+  include Msf::Exploit::FileDropper
+
+  def initialize(info = {})
+    super(
+      update_info(
+        info,
+        'Name' => 'Car Rental System 1.0 reverse shell',
```

Comment:
Yeah, this shouldn't always be a reverse shell, and this title should describe what your exploiting. In this case it appears to be a SQL injection for authentication bypass followed by an arbitrary file upload vulnerability.